app/controllers/api/v2/hosts_controller.rb

```diff
@@ -37,6 +37,12 @@ class HostsController < V2::BaseController
       def index
         @hosts = action_scope_for(:index, resource_scope_for_index)
 
+        if params[:thin]
+          @hosts = @hosts.reorder(:name).distinct.pluck(:id, :name)
```

Comment:
we have to reorder on a field we have, if user passed an order param on something that isn't in the query it will fail.